packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -130,10 +131,16 @@ class MethodChannel {
   /// The message codec used by this channel, not null.
   final MethodCodec codec;
 
+<<<<<<< HEAD
```

Comment:
Bad merge.  Looks like a lot of stuff got lost too.

Reply:
Ya... still familiarizing myself with git... Trying to recover the commits..

Reply:
Phew... got everything back!